src/module/item/condition/persistent-damage-editor.ts

```diff
@@ -1,30 +1,44 @@
 import type { ActorPF2e } from "@actor";
+import type { ApplicationConfiguration, ApplicationRenderContext } from "@client/applications/_module.mjs";
+import type { HandlebarsRenderOptions } from "@client/applications/api/_module.mjs";
 import { damageDiceIcon } from "@system/damage/helpers.ts";
 import { DamageRoll } from "@system/damage/roll.ts";
-import { DamageType } from "@system/damage/types.ts";
+import type { DamageType } from "@system/damage/types.ts";
 import { DAMAGE_TYPE_ICONS } from "@system/damage/values.ts";
 import { htmlClosest, htmlQuery, htmlQueryAll } from "@util";
 import * as R from "remeda";
 import type { PersistentDamagePF2e } from "./document.ts";
-import appv1 = foundry.appv1;
 
-class PersistentDamageDialog extends appv1.api.Application<PersistentDamageDialogOptions> {
-    constructor(
-        private actor: ActorPF2e,
-        options: Partial<PersistentDamageDialogOptions> = {},
-    ) {
-        super(options);
-        actor.apps[this.appId] = this;
-    }
-
-    static override get defaultOptions(): appv1.api.ApplicationV1Options {
-        return {
-            ...super.defaultOptions,
-            classes: ["persistent-damage-dialog"],
-            template: "systems/pf2e/templates/items/persistent-damage-dialog.hbs",
+class PersistentDamageEditor extends fa.api.HandlebarsApplicationMixin(fa.api.ApplicationV2) {
+    static override DEFAULT_OPTIONS: DeepPartial<fa.ApplicationConfiguration> = {
```

Comment:
Need at least a title, probably also a `contentClasses` of `["standard-form"]`

Reply:
There is already a title, though its a getter so that the actor's name is included.

Are you certain about standard-form? It lets me remove the gaps, but it looks off

![image](https://github.com/user-attachments/assets/dfdf5d43-e9bf-4f97-8acf-5dfe80232110)

Reply:
With HR reintroduced

![image](https://github.com/user-attachments/assets/ddf5c355-7652-40eb-bc96-d99a0da05b9c)

With Bonus Tap Padding

![image](https://github.com/user-attachments/assets/d5458cb5-cfcd-4d29-aede-bc3323327d70)

CHOOSE (the one from previous post is valid too)